examples/from_googlesheets.py

```diff
@@ -0,0 +1,27 @@
+"""Example of using PandasAI with GoogleSheets"""
+
+import pandas as pd
+from data.sample_dataframe import dataframe
+
+from pandasai import PandasAI
+from pandasai.helpers.datasource import from_google_sheets
+from pandasai.llm.openai import OpenAI
+
+# instructions: https://docs.google.com/document/d/1uAmXI3e8Wm26lZQb0bYIJaXRdZi5WLJKNJHT59QMGEc/edit?usp=sharing
+
+
+df = from_google_sheets(
+    "1IRGJFUpCR0-9tPDk2MgPjQCtNft_4a8B1fHgYK7_LM8",
+    "second",
+    "../service_account_file.json",
```

Comment:
I would probably set this as an optional step with a default value (which could be `google_service_account_file.json` by default. What do you think?